students/Chao/project_fpy/fpy.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
+
+"""
+Project Fpy
+=================
+Po-Sung (Sean) Chao
+
+The purpose of this Python program is to build a maintenance tool with F5 Python SDK
+
+Package required: F5 Python SDK
+$ pip install f5-sdk
+=================
+"""
+
+from f5.bigip import ManagementRoot
+from f5.utils.responses.handlers import Stats
+from icontrol.exceptions import iControlUnexpectedHTTPError
+from requests.exceptions import ConnectionError
+import getpass
+import ipaddress
+
+class LTM:
+    """ Main Class for Local Traffic Manager Module """
+    def __init__(self, mgmt):
+        self.mgmt = mgmt
+
+class Pool(LTM):
+    """ Pool class for retrieving pool status """
+
+    def get_pool_state(self, p_name, p_partition='Common'):
+        """ Function to retrieve current pool availability status """
+        pool_stat = self.mgmt.tm.ltm.pools.pool.load(name=p_name, partition=p_partition)
+        stats = Stats(pool_stat.stats.load())
+        return stats.stat.status_availabilityState['description']
+
+    def get_pool(self):
```

Comment:
Since you aren't actually printing anything, better to call this print_pool_status or something like that.